frontend/stylesheets/trace_view/_status_bar.less

```
@@ -3,7 +3,14 @@
   margin-right: 25px;
   font-size: 14px;
   display: flex;
-  flex-wrap: wrap;
+  // Kept as nowrap (rather than wrap), and with white-space: nowrap below, so that
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)